code/game/gamemodes/vampire/vampire.dm

```diff
@@ -297,22 +297,23 @@ You are weak to holy things and starlight. Don't go into space and avoid the Cha
 			to_chat(owner, "<span class='warning'>They've got no blood left to give.</span>")
 			break
 		if(H.stat < DEAD)
-			if(!issmall(H) && !H.ckey)
```

Comment:
did you try it with just && H.ckey cause optimally it needs to stay as is but the check needs to read is "As long as its not small and has a ckey" if you leave it as an || its gonna pick up on monkeys with ckeys as currently it reads "As long as not small OR has a ckey". If one of those conditions is met its gonna fire the code. If its not small its gonna fire. If it has a ckey its gonna fire.

Reply:
see line 309:
`// not small OR has a ckey, monkeys with ckeys can be sucked, humanized monkeys can be sucked monkeys without ckeys cannot be sucked
 `

Reply:
This is how it's been prior to the PR, you've always been able to suck monkeys with ckeys

Reply:
they should be able to gain powers from it however. the PR that caused this issue put a block on it. Just food.
From the orignal PR: "Blood from monkey mobs: Vampires are able to now get blood from mokey mobs, such as from monkeys or stoks, for sustenance. HOWEVER they do NOT get any blood points to count towards their vampire powers."

Your changes remove that block. It needs to stay. line 316 is handling the food bit.

Reply:
hold on rethinking now that i am getting some caffine in me. The logic seems to be off almost. We don't want them getting power from ckeyless mobs even if humanized so i would double check that.

Reply:
Prior to the latest merged vampire PR(the one this is hotfixing) you were capable of gaining powers from monkeys that had ckeys I just restored that functionality because I don't believe that it was intentionally removed

Reply:
again just double check its not unintionally givieng powers from ckeyless ones monkey or humanized

Reply:
https://github.com/ParadiseSS13/Paradise/blob/fd8a9e4909272b911854da49c446139f1171693e/code/modules/mob/living/carbon/human/species/_species.dm#L344

Reply:
I only half trust DM procs to work as logically intended from experience though i how the other proc would stop it yes. I just err on the side of over paranoia as there is nothing logical about Dreammaker procs...Ever. I am now having PTSD.

Reply:
It worked in my tests and the logic makes sense, if no ckey and small(monkey) then no power gain, if ckey and small, power gain, if no ckey and humanized, power gain, if ckey and humanized, power gain.

Reply:
@Aurorablade 
![image](https://user-images.githubusercontent.com/30060146/60934919-01c51c80-a296-11e9-8fde-b3da2a12f5c5.png)

Reply:
Alright. yeah again just over paranoia. Espically after that one time invovling some missing organs...and yes it was something i was codeing and testing.